doc/man/8/ceph-disk.rst

```diff
@@ -15,31 +15,30 @@ Synopsis
                 ...
 
 | optional arguments:
-  -h, --help            show this help message and exit
-  -v, --verbose         be more verbose
-  --log-stdout          log to stdout
-  --prepend-to-path PATH
+|   -h, --help          show this help message and exit
+|   -v, --verbose       be more verbose
+|   --log-stdout        log to stdout
+|   --prepend-to-path PATH
                         prepend PATH to $PATH for backward compatibility (default /usr/bin)
-  --statedir PATH       directory in which ceph state is preserved (default /var/lib/ceph)
-  --sysconfdir PATH     directory in which ceph configuration files are found (default /etc/ceph)
-  --setuser USER        use the given user for subprocesses, rather than ceph or root
-  --setgroup GROUP      use the given group for subprocesses, rather than ceph or root
+|   --statedir PATH     directory in which ceph state is preserved (default /var/lib/ceph)
+|   --sysconfdir PATH   directory in which ceph configuration files are found (default /etc/ceph)
+|   --setuser USER      use the given user for subprocesses, rather than ceph or root
+|   --setgroup GROUP    use the given group for subprocesses, rather than ceph or root
```

Comment:
It used to be, changed format refer to ceph-disk --help, the commit is:

commit d355a383d17d9ad1dd249a2760da585d4965649d
Author: Loic Dachary <ldachary@redhat.com>
Date:   Thu Feb 25 17:20:54 2016 +0700

    doc: update ceph-disk to refer to ceph-disk --help
    
    The ceph-disk page is often obsolete, mostly because maintaining
    it requires a significant amount of copy/paste and re-formatting.
    Now that the --help of ceph-disk has been updated to include a more
    verbose explanation of each subcommand, simplify the man page to
    give an overview of the subcommands and suggest the --help is used
    to get more information.
    
    Signed-off-by: Loic Dachary <loic@dachary.org>